site/content/en/latest/tasks/observability/gateway-exported-metrics.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Gateway Exported Metrics"
```

Comment:
can we call this Envoy Gateway Metrics or Control Plane Metrics ?

Reply:
How about `Envoy Gateway Metrics List`? The `Envoy Gateway Metrics` or `Control Plane Metrics` may be confusing, we need a name to tell whether this doc is for usage or for metrics list.

Reply:
sure that works too, merging this PR for now